spec/support/page_objects/publish/courses/new_outcome_page.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Publish
+    module Courses
+      class NewOutcome < PageObjects::Base
+        set_url "/publish/organisations/{provider_code}/{recruitment_cycle_year}/courses/outcome/new{?query*}"
+
+        section :qualification_fields, '[data-qa="course__qualification"]' do
+          element :qts,           "#course_qualification_qts"
```

Comment:
weird tab spacing

Reply:
fixed in https://github.com/DFE-Digital/teacher-training-api/pull/2471/commits/70f5a9167981cc9e75c455c54d1a694b6cd139d6